pkg/chart/airflow.go

```diff
@@ -381,7 +381,12 @@ func (c Client) createAirflowDatabase(ctx context.Context, teamID, dbPassword st
 	}
 
 	dbInstance := createAirflowcloudSQLInstanceName(teamID)
-	if err := createCloudSQLInstance(ctx, dbInstance, c.gcpProject, c.gcpRegion); err != nil {
+	team, err := c.repo.TeamGet(ctx, teamID)
```

Comment:
Hvis du tar en titt på funksjonen som kaller på denne koden, så ser du at den bruker `team.ID`, så da kan du heller bare sende inn teamet, i stedet for å gjøre et nytt databasekall.

Reply:
yes!